thunder/core/prims.py

```diff
@@ -3187,6 +3188,7 @@ def transpose_meta(a: TensorProxy, /, permutation: tuple[int, ...]) -> TensorPro
 
 
 view = make_prim(PrimIDs.VIEW, "view", meta=reshape_meta, tags=(OpTags.SHAPE_OP,))
+view_as = make_prim(PrimIDs.VIEW_AS, "view_as", meta=reshape_meta, tags=(OpTags.SHAPE_OP,))
```

Comment:
Let's not add a new primitive -- it won't be called by the implementation, anyway

Reply:
Just for my understanding, when do we have to add a new primitive? Is there a general rule of thumb?